packages/tmate-ubus/Makefile

```diff
@@ -0,0 +1,31 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tmate-ubus
```

Comment:
All other ubus packages follow the naming convention ubus-* .We should do the same for this one.